### ISSUE TITLE
Request: Textured statusbars

### ISSUE THREAD

**Issue by greevir** [powercrystals/MineFactoryReloaded#156]
I'd like to request textured status bars (power, work, and idle) to be textured instead of hard-coded. I imagine it would make some texture artists happy. (and me too) Thanks!

**Comment by powercrystals** [comment]
Would you be happy if I just had a single set of bars used by all GUIs? I'd really rather not have to do that to every GUI since there's like 30.

**Comment by greevir** [comment]
Yeah, that'd be great. It's kind of what I was expecting anyways.
